src/qibo/tests/test_qasm.py

```diff
@@ -32,6 +35,17 @@ def test_simple():
     assert_strings_equal(c.to_qasm(), target)
 
 
+def test_simple_cirq_qiskit(atol=1e-7):

```

Comment:
Is there any reason we mention `qiskit` here? We are not testing anything with qiskit right?